File: src/main/kotlin-graphql/io/github/graphql/kotlin/model/RepositoryReleaseParametrizedInput.kt
package io.github.graphql.kotlin.model

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLParametrizedInput
import java.util.StringJoiner
/**
 * Parametrized input for field release in type Repository
 */
@javax.annotation.Generated(
    value = ["com.kobylynskyi.graphql.codegen.GraphQLCodegen"],
    date = "2021-03-07T19:48:13+0800"
)
data class RepositoryReleaseParametrizedInput(
    val tagName: String
) : GraphQLParametrizedInput {

    override fun toString(): String {
        val joiner = StringJoiner(", ", "( ", " )")
        joiner.add("tagName: " + GraphQLRequestSerializer.getEntry(tagName))
        return joiner.toString()
    }
}

File: src/main/scala-graphql/io/github/graphql/model/RepositoryReleaseParametrizedInput.scala
package io.github.graphql.model

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLParametrizedInput
import scala.collection.JavaConverters._

/**
 * Parametrized input for field release in type Repository
 */
@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2021-03-06T19:48:00+0800"
)
case class RepositoryReleaseParametrizedInput(
    @javax.validation.constraints.NotNull
    tagName: String
) extends GraphQLParametrizedInput {

    override def toString(): String = {
        Seq(
            if (tagName != null) "tagName: " + GraphQLRequestSerializer.getEntry(tagName) else ""
        ).filter(_ != "").mkString("(", ",", ")")
    }

}
